packages/components/src/tip/index.js

```diff
@@ -4,21 +4,42 @@
 import { SVG, Path } from '@wordpress/primitives';
 
 /**
- * @typedef Props
- * @property {import('react').ReactNode} children Children to render in the tip.
+ * Internal dependencies
  */
+import useShadowStyles, { css } from '../utils/hooks/use-shadow-styles';
+import { ALERT, COLORS } from '../utils/colors-values';
+import { space } from '../utils/space';
+
+const styles = css`
+	:host {
+		display: flex;
+		color: ${ COLORS.gray[ 700 ] };
+	}
+
+	::slotted( svg ) {
+		align-self: center;
+		fill: ${ ALERT.yellow };
+		flex-shrink: 0;
+		margin-right: ${ space( 2 ) };
+	}
+
+	::slotted( p ) {
+		margin: 0;
+	}
+`;
 
 /**
- * @param {Props} props
+ * @param {Object} props
+ * @param {import('react').ReactNode} props.children Children to render in the tip.
  * @return {JSX.Element} Element
  */
-function Tip( props ) {
+function Tip( { children } ) {
 	return (
-		<div className="components-tip">
+		<div ref={ useShadowStyles( null, [ styles ] ) }>
```

Comment:
We can keep still use `className="components-tip"` for debugging, discovery, or third-party/external styling. As well we could change the element's name from `<div>` to something more semantic like `<wp-tip>`